opentracing/tracer.py

```diff
@@ -115,62 +115,76 @@ def extract(self, format, carrier):
         return self._noop_span_context
 
 
-class ReferenceType:
+class ReferenceType(object):
     """A namespace for OpenTracing reference types.
 
-    See http://opentracing.io/spec for more detail about references, reference
-    types, and CHILD_OF and FOLLOWS_FROM in particular.
+    See http://opentracing.io/spec for more detail about references,
+    reference types, and CHILD_OF and FOLLOWS_FROM in particular.
     """
     CHILD_OF = 'child_of'
     FOLLOWS_FROM = 'follows_from'
 
 
-# We inherit from namedtuple (rather than assign) so that we can specify a
-# standard docstring.
-class Reference(namedtuple('Reference', ['type', 'span_context'])):
+# We use namedtuple since references are meant to be immutable.
+# We subclass it to expose a standard docstring.
+class Reference(namedtuple('Reference', ['type', 'referee'])):
```

Comment:
(need not be part of this PR, and if you agree I am happy to make the change here and in `basictracer-py`)